rr_iarrc/launch/line_detector_segnet.launch

```diff
@@ -0,0 +1,7 @@
+<launch>
+    <include file="$(find rr_ml)/launch/run_seg.launch">
+        <arg name="image_topic" value="/camera/image_color_rect"/>
+        <arg name="detection_topic" value="/lines_detection_img"/>
+        <arg name="model" value="iarrc_lines"/>
```

Comment:
Does this assume a specific path?

Reply:
Kind of. There is a specific folder in which it looks for iarrc_lines.h5. That is set in run_seg.launch